content/exporters/supported-exporters/Python/Honeycomb.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Honeycomb.io (Tracing)"
+date: 2010-03-18
+draft: false
+weight: 3
+class: "resized-logo"
+aliases: [/guides/exporters/supported-exporters/python/honeycomb]
+logo: /img/honeycomb-logo.jpg
+---
+
+- [Introduction](#introduction)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your traces](#viewing-your-traces)
+
+## Introduction
+
+[Honeycomb](https://www.honeycomb.io) is a service for debugging your software in production. Capture traces or individual events, then speedily slice and dice your data to uncover patterns, find outliers, and understand historical trends.
+
+Its OpenCensus Python exporter is available at [https://github.com/codeboten/ochoneycomb](https://github.com/codeboten/ochoneycomb)
```

Comment:
Perhaps let's use the pip install here and then put the URL as a reference.
So how about

\## Installing it
The Honeycomb Python exporter can be installed from pip by
\```shell
pip install ochoneycomb
\```


and then at the bottom add

\## References
Resource|URL
---|---
Honeycomb exporter on pip|https://pypi.org/project/ochoneycomb/
Honeycomb exporter on Github|https://github.com/codeboten/ochoneycomb